README.md

```diff
@@ -0,0 +1,29 @@
+# Discord
+
+Discord is a free all-in-one messaging, voice and video client that's available on your computer and phone.
+
+This repo hosts the flatpak wrapper for [Discord](https://discord.com/), available at [Flathub](https://flathub.org/apps/details/com.discordapp.Discord).
+
+```sh
+flatpak install flathub com.discordapp.Discord
+flatpak run com.discordapp.Discord
+```
+
+
+## Differences in flatpak version
+
+The flatpak version runs in a sandbox to provide better safety and privacy for users (see [this privacy report](https://spyware.neocities.org/articles/discord.html)).
+
+However, this sandboxing prevents the following features from working:
+
+- **Game Activity**: This flatpak version of Discord cannot scan running processes to detect running games.  
+  There is currently no workaround or solution for this limitation.
+- **Unrestricted File Access**: Default sandbox permissions limit Discord to only certain directories, so you can't access your entire Home directory. This limits which file directories you can attach files from and impacts drag and drop functionality. You can change sandbox permissions of installed flatpak applications (for example, with [Flatseal](https://flathub.org/apps/details/com.github.tchx84.Flatseal)) to give Discord broader file system access, allowing file attachments from more locations.
```

Comment:
I would mention here that the plan is for Zypak to give us a file picker portal which will allow full access to the filesystem while still restricting unauthorized access.

Reply:
What would be the flatpak override command to allow access to Home? Is it `flatpak override --filesystem=home com.discordapp.Discord` ? Should I add this command as an alternative to Flatseal? @lionirdeadman

Reply:
```suggestion
- **Unrestricted File Access**: Default sandbox permissions limit Discord to only certain directories, so you can't access your entire Home directory. Currently, this limits which file directories you can attach files from and impacts drag and drop functionality.  
  This limitation will likely be overcomed eventually, when Zypak give us a file picker portal which will allow full access to the filesystem while still restricting unauthorized access.  
  To work around this now, you can change sandbox permissions of installed flatpak applications (for example, with [Flatseal](https://flathub.org/apps/details/com.github.tchx84.Flatseal) or with `flatpak override --filesystem=home com.discordapp.Discord`) to give Discord broader file system access, allowing file attachments from more locations.
```

Reply:
It needs --user if it's installed as user application rather than system.

Reply:
Well it's just an example anyways. It might also need sudo I think, if it's installed on system, but you'd get an error and know what to do. Feel free to rephrase it if needed.

Reply:
Oh, okay. I didn't know you could add --user overrides on system installed apps too. I'll change it then.

Reply:
You can't add --user to system installed apps. System and user applications have different permission overrides. Confusing, I know.

Reply:
Are you sure? I just tested with a system installed Discord flatpak (confirmed with `flatpak info com.discordapp.Discord`). I did `flatpak override --user ...`, `flatpak info --show-permissions ...` and then again with `su otheruser` and it does show the overridden home permission for just one user.

Reply:
Though maybe we should change it to `home:ro` if the purpose is only add attachments.
I didn't test this in the actual app by the way, whether just adding the `home` or `home:ro` permission actually allows file attachments properly. I'm assuming this works by what you guys have discussed.

Reply:
Your flatpak info is for the system one, it also needs to be specificed --user.

`home:ro` would definitely work but I think realistically that zypak should get out of beta probably soon so it's not a mess I'm ready to make.

Reply:
That's the thing, there's no user installation (`flatpak info --user ...` fails, with and without --show-permissions), I only have the default system installation for Discord.
Yet the --show-permissions shows the home permission only on the user that added it (with override --user). I haven't tested it, but this makes it seem that you **can** override it per user even on system wide installation. Either this is documented confusingly [here](https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak-override), or maybe it's some kind of bug...?